test/Test/Main.purs

```diff
@@ -424,6 +434,17 @@ test_parallel_throw = assert "parallel/throw" $ withTimeout (Milliseconds 100.0)
   r2 ← readRef ref
   pure (isLeft r1 && r2 == "foo")
 
+test_parallel_throw_invincible ∷ Aff Unit
+test_parallel_throw_invincible = assert "parallel/throw/invincible" do
+  map (all identity) $ parSequence
+    [ false <$ do
+        invincible $ delay $ Milliseconds 50.0
+    , false <$ do
+        delay $ Milliseconds 10.0
+        throwError $ error "crash"
+    ] `catchError` \e ->
+      pure $ [ Error.message e == "crash" ]
+
```

Comment:
Similar to the other test, I'm not sure what this is verifying other than that you've included an `invincible` which presumably triggers a runtime error. I would still like to verify some sort of behavior around `invincible`, since that is valuable in specifying what the behavior should be. So in the case of `parSequence`, I would expect the `invincible` block to fully complete (which I would check by modifying a Ref), but then still observe the exception.